src/_posts/hands-on-i.md

```diff
@@ -0,0 +1,149 @@
+
+```html
+<a-box id="circuitboard" position="0 0.0115 0.4"
+       width="0.05" depth="0.05" height="0.003" color="#008000">
+  <a-box id="chip" position="0 0.002 0.005"
+         width="0.02" depth="0.02" height="0.001" color="#000">
+  </a-box>
+</a-box>
+```
+
+A-Frame provides a way of adding empty nodes to the scene or, in A-Frame nomenclature, an **entity**. [Entities](/docs/0.6.0/core/entity.html) are represented by the `a-entity` element and they have nothing associated with them: no geometry, model, or material. They do have implicit `rotation`, `position`, `scale` and `visible` attributes, which are automatically injected by the framework for all the elements within the scene.
+
+Adding the [`hand-controls`](/docs/0.6.0/components/hand-controls.html) attribute to an entity displays a hand model following the position and orientation of one of the tracked controllers of an HTC Vive or Oculus.
+
+```html
+<a-entity id="left-hand" hand-controls="left"></a-entity>
+<a-entity id="right-hand" hand-controls="right"></a-entity>
+```
+
+This is the resulting scene:
+
+[![Initial scene](/images/blog/hands-on/step1-initial-scene.png)](/images/blog/hands-on/step1-initial-scene.png)
+
+### One workspace to rule them all
+
+I was developing in front of a desk so for testing purposes I placed the elements as if they were resting on my real desktop only 5cm higher up. This allowed me to avoid my hands colliding  with my physical keyboard when I’m in VR mode. Adjusting the whole workspace at the same time is achieved by changing the `position` attribute of the `workspace` entity. Thanks to the real and virtual table heights matching, the demo has a great feeling of presence.
+
+A simpler alternative would have been to make the circuit board and tools children of the table. Unfortunately this configuration does not play well when adding physics during the next article. The physics engine would consider the nested elements to be compounding parts of the root element and I didn’t want my tools and circuit board to be considered part of the table but separated elements.
+
+Also notice that if you are experiencing room-scale VR, you will also need to position the workspace and camera relative to the center of your configured stage, making them match with your real-life table and yourself respectively.
+
+Finally, while developing I spend most of my the time sitting in front of my computer. So as a last touch I changed the height correction of the camera, the `user-height` attribute, to match the distance of my eyes from the floor when I'm sitting down:
```

Comment:
This for 2D mode right? Because user height is ignored when entering VR with a headset